apps/ensrainbow/src/index.ts

```diff
@@ -24,11 +26,7 @@ try {
 }
 
 app.get("/v1/heal/:labelhash", async (c: Context) => {
-  const labelhash = c.req.param("labelhash")?.toLowerCase();
-
-  if (!labelhash) {
-    return c.json({ error: "Missing labelhash parameter" }, 400);
-  }
+  const labelhash = c.req.param("labelhash").toLowerCase();
 
   if (!/^(0x)?[0-9a-f]{64}$/.test(labelhash)) {
```

Comment:
let's replace this fn with `isHex(labelhash) && size(labelhash) === 32` to remove our custom regex

(`import { isHex, size } from 'viem'`)

Reply:
likewise below, can you see if `viem#hexToBytes` can replace the line like

```
const hashBytes = Buffer.from(labelhash.replace(/^0x/, ""), "hex");
```

Reply:
@lightwalker-eth What do you think? Do you want logs about unhealable labelhashes on production?